.platform/nginx/conf.d/elasticbeanstalk/01_admin-access.conf

```
@@ -2,18 +2,28 @@ set_real_ip_from  0.0.0.0/0;
 real_ip_header    X-Forwarded-For;
 real_ip_recursive on;
 
+proxy_http_version  1.1;
+
+proxy_set_header    Connection          $connection_upgrade;
+proxy_set_header    Upgrade             $http_upgrade;
+proxy_set_header    Host                $host;
+proxy_set_header    X-Real-IP           $remote_addr;
+proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
+
 location /admin {
   allow 121.165.253.64/27;
   allow 121.165.253.192/27;
   allow 58.236.10.80/29;
   deny  all;
 
   proxy_pass          http://127.0.0.1:5000;
-  proxy_http_version  1.1;
+}
 
-  proxy_set_header    Connection          $connection_upgrade;
-  proxy_set_header    Upgrade             $http_upgrade;
-  proxy_set_header    Host                $host;
-  proxy_set_header    X-Real-IP           $remote_addr;
-  proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
+location /docs {
+  allow 121.165.253.64/27;
+  allow 121.165.253.192/27;
+  allow 58.236.10.80/29;
+  deny  all;
+
```

a: 아래와 같이 설정을 추가하면 `/docs`로 접근 시 `/docs/index.html`을 받을 수 있을 것 같습니다~

참고: http://nginx.org/en/docs/http/ngx_http_index_module.html

애플리케이션 레벨에서 처리하는 방법도 있겠지만, 단순 문서에 대한 처리라 방법은 상관은 없을 것 같아요.

```suggestion

  index  index.html;
  
```